src/Seksjon/Sertifikat.elm

```diff
@@ -0,0 +1,1323 @@
+
+                VisOppsummeringEtterEndring validertSertifikatSkjema ->
+                    gåTilEndreSkjema model validertSertifikatSkjema
+
+                _ ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+        SkjemaEndret skjemaEndring ->
+            case model.aktivSamtale of
+                EndreOpplysninger sertifikatSkjema ->
+                    ( sertifikatSkjema
+                        |> oppdaterSkjema skjemaEndring
+                        |> EndreOpplysninger
+                        |> oppdaterSamtalesteg model
+                    , Cmd.none
+                    )
+                        |> IkkeFerdig
+
+                _ ->
+                    ( Model model, Cmd.none )
+                        |> IkkeFerdig
+
+        VilLagreEndretSkjema ->
+            case model.aktivSamtale of
+                EndreOpplysninger skjema ->
+                    case SertifikatSkjema.valider skjema of
+                        Just validertSkjema ->
+                            ( validertSkjema
+                                |> VisOppsummeringEtterEndring
+                                |> nesteSamtaleSteg model (Melding.svar (validertSkjemaTilSetninger validertSkjema))
+                            , Cmd.batch
+                                [ lagtTilSpørsmålCmd model.debugStatus
+                                ]
+                            )
+                                |> IkkeFerdig
+
+                        Nothing ->
+                            ( skjema
+                                |> SertifikatSkjema.visAlleFeilmeldinger
+                                |> EndreOpplysninger
+                                |> oppdaterSamtalesteg model
+                            , Cmd.none
+                            )
+                                |> IkkeFerdig
+
+                _ ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+        VilLagreSertifikat ->
+            case model.aktivSamtale of
+                VisOppsummering skjema ->
+                    IkkeFerdig
+                        ( skjema
+                            |> LagrerSkjema
+                            |> nesteSamtaleSteg model (Melding.svar [ "Ja, informasjonen er riktig" ])
+                        , Cmd.batch
+                            [ postEllerPutSertifikat SertifikatLagret skjema
+                            , lagtTilSpørsmålCmd model.debugStatus
+                            ]
+                        )
+
+                VisOppsummeringEtterEndring skjema ->
+                    IkkeFerdig
+                        ( skjema
+                            |> LagrerSkjema
+                            |> nesteSamtaleSteg model (Melding.svar [ "Ja, informasjonen er riktig" ])
+                        , Cmd.batch
+                            [ postEllerPutSertifikat SertifikatLagret skjema
+                            , lagtTilSpørsmålCmd model.debugStatus
```

Comment:
Samme her, og disse to tilfellene er jo helt like, så du kan godt trekke dem ut som en funksjon som du kaller fra både `VisOppsummering` og `VisOppsummeringEtterEndring`